### PR TITLE
Improve Issue type "support" message

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Improve Issue type "support" message with links to forum, slack and stack overflow
| Type?         | improvement
| Category?     | PM
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19567)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Matt75** [comment]
Maybe we should recommend to use Addons free Support for Modules ?

If question or issue concern a module can be found on Addons, please contact the module developer with link provided on Addons module page.

For example : Ebay, PayPal, PrestaShop Checkout etc...

You can find some issues by searching by module name

**Comment by matks** [comment]
Also nowhere here do we talk about the User Documentation and Developer Documentation 🤔

**Comment by matks** [comment]
PR updated

**Review by PierreRambaud — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+- PrestaShop user documentation http://doc.prestashop.com/display/PS17/User+Guide
+- PrestaShop developer documentation http://devdocs.prestashop.com/
+
+You can also discuss with other PrestaShop users on
+- community forums (https://www.prestashop.com/forums/)
+- Stack Overflow (https://stackoverflow.com/questions/tagged/prestashop)
+- community Slack channel (https://github.com/PrestaShop/open-source/blob/master/slack/readme.md)
+
+
+If you have questions about a PrestaShop you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.
```
Comment (NeOMakinG) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
If you have questions about a PrestaShop's module/theme you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.
```

Isn't it better?

Comment (matks) [reply .github/ISSUE_TEMPLATE/3_support_question.md]: Yes it is 😄

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+about: For questions about using PrestaShop
 
 ---
 
 We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.
-If you have questions about using PrestaShop or third-party modules, or if you need help
-with your shop, please refer to one of our support plans: https://www.prestashop.com/support
+If you have questions about using PrestaShop, you might find the answer in
+- PrestaShop user documentation http://doc.prestashop.com/display/PS17/User+Guide
+- PrestaShop developer documentation http://devdocs.prestashop.com/
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion

If you have questions about using PrestaShop, you might find the answer here:
- User's guide - http://doc.prestashop.com/display/PS17/User+Guide
- Developer documentation - https://devdocs.prestashop.com/

If you need help with your shop or third-party modules, please consider one of our support plans: https://www.prestashop.com/support
```

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 3]
```diff
@@ -1,11 +1,24 @@
...
+
+You can also discuss with other PrestaShop users on
+- community forums (https://www.prestashop.com/forums/)
+- Stack Overflow (https://stackoverflow.com/questions/tagged/prestashop)
+- community Slack channel (https://github.com/PrestaShop/open-source/blob/master/slack/readme.md)
+
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
If you have questions about a PrestaShop's module/theme you have downloaded or bought on the PrestaShop Addons Marketplace, you can contact the module developer through the link provided on Addons module page.

Alternatively, you can also ask for help from other PrestaShop users:
- Community forums: https://www.prestashop.com/forums/
- Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
- Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md

```

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 4]
```diff
@@ -1,11 +1,24 @@
...
+
+If you have questions about a PrestaShop's module/theme you have downloaded or bought on PrestaShop Addons marketplace, you can contact the module developer through the link provided on Addons module page.
+
+
+If you need help with your shop or third-party modules, you can also refer to one of our support plans: https://www.prestashop.com/support
+
```
Comment (eternoendless) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
```

**Review by eternoendless — COMMENTED** [review]
Consider changing the order of paragraphs

**Comment by matks** [comment]
Applied @eternoendless feedback 👍

**Review by atomiix — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 5]
```diff
@@ -1,11 +1,22 @@
...
 
 We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.
-If you have questions about using PrestaShop or third-party modules, or if you need help
-with your shop, please refer to one of our support plans: https://www.prestashop.com/support
+
+If you have questions about using PrestaShop, you might find the answer here:
+- User's guide - http://doc.prestashop.com/display/PS17/User+Guide
+- Developer documentation - https://devdocs.prestashop.com/
+
+If you need help with your shop or third-party modules, please consider one of our support plans: https://www.prestashop.com/support
```
Comment (Progi1984) [inline .github/ISSUE_TEMPLATE/3_support_question.md]: I have problem with our : who is `we` ? The Open Source Project ? The PrestaShop Company ? The Maintainers Team ?

File: .github/ISSUE_TEMPLATE/3_support_question.md [hunk 6]
```diff
@@ -1,11 +1,22 @@
...
+
+If you have questions about a PrestaShop's module/theme you have downloaded or bought on the PrestaShop Addons Marketplace, you can contact the module developer through the link provided on Addons module page.
+
+Alternatively, you can also ask for help from other PrestaShop users:
+- Community forums: https://www.prestashop.com/forums/
+- Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
+- Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md
```
Comment (Progi1984) [inline .github/ISSUE_TEMPLATE/3_support_question.md]:
```suggestion
- Community chat: https://github.com/PrestaShop/open-source/blob/master/slack/readme.md
- Community forums: https://www.prestashop.com/forums/
- Stack Overflow: https://stackoverflow.com/questions/tagged/prestashop
```

**Review by Progi1984 — COMMENTED** [review]
A question

File: .github/ISSUE_TEMPLATE/3_support_question.md (hunk 5 above)
Comment (matks) [reply .github/ISSUE_TEMPLATE/3_support_question.md]: PS company

Comment (Progi1984) [reply .github/ISSUE_TEMPLATE/3_support_question.md]: In this case, the `our` must be contextualized

Comment (matks) [reply .github/ISSUE_TEMPLATE/3_support_question.md]: `$this->context->we` ?

Comment (Progi1984) [reply .github/ISSUE_TEMPLATE/3_support_question.md]: :trollface: 

```suggestion
If you need help with your shop or third-party modules, please consider the PrestaShop Company support plans: https://www.prestashop.com/support
```

**Review by matthieu-rolland — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]

**Comment by matks** [comment]
Applied @Progi1984 suggestion so I removed all your approvals, folks ! 😛

**Review by matthieu-rolland — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**